usaspending_api/common/query_with_filters.py

```diff
@@ -525,7 +493,9 @@ class QueryWithFilters:
     unsupported_filters = ["legal_entities"]
 
     @classmethod
-    def _generate_elasticsearch_query(cls, filters: dict, query_type: _QueryType, nested_path: str = "") -> ES_Q:
+    def _generate_elasticsearch_query(cls, filters: dict, query_type: _QueryType, **options) -> ES_Q:
+        nested_path = options.pop("nested_path", "")
```

Comment:
Why is the option retrieved with a `pop` here, whereas the other options use a `get`?